build.gradle

```diff
@@ -400,3 +421,5 @@ task genJaxbAddCase {
 }
 
 compileJava.dependsOn genJaxb
+compileJava.dependsOn genJaxbAddCase
+compileJava.dependsOn genJaxbAddLogEntry
```

Comment:
Do we need to regenerate the Jaxb classes everytime we build. Don't we still move them into java/source and commit them to the repo?